sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -886,22 +892,26 @@ export class ServiceBusMessageImpl implements ServiceBusReceivedMessage {
       this.lockToken = undefined;
     }
 
-    let actualBodyType:
-      | ReturnType<typeof defaultDataTransformer["decodeWithType"]>["bodyType"]
-      | undefined = undefined;
+    // let actualBodyType:
+    //   | ReturnType<typeof defaultDataTransformer["decodeWithType"]>["bodyType"]
+    //   | undefined = undefined;
 
-    if (msg.body) {
-      try {
-        const result = defaultDataTransformer.decodeWithType(msg.body);
+    // if (msg.body) {
+    //   try {
+    //     const result = defaultDataTransformer.decodeWithType(msg.body);
+
+    //     this.body = result.body;
+    //     actualBodyType = result.bodyType;
+    //   } catch (err) {
+    //     this.body = undefined;
+    //   }
+    // }
+    // why above when `fromRheaMessage()` already called `defaultDataTransformer.decodeWithType()` earlier on message body
```

Comment:
Can you sync with @richardpark-msft about this?

Reply:
Discussed with @richardpark-msft and we agreed that this code was doing double-decoding.  So deleting.